src/viser/_message_api.py

```diff
@@ -207,6 +213,44 @@ def configure_theme(
             ),
         )
 
+    def add_glb(
+        self,
+        name,
+        glb_path,
```

Comment:
I'd vote to just take `bytes` directly as input here, and leave the user responsible for figuring out where they come from

(it would also enable `trimesh => bytes => add_glb()` without writing to a file)